packages/dashboard/README.md

```diff
@@ -4,6 +4,16 @@ This package is work in progress.
 
 This package contains a Next.js app for sidetree nodes.
 
+### Add a Custom Logo to the Dashboard
+
+Create a file named `.evt.local` in this directory (next to `package.json`),
```

Comment:
don't do this... use the config that already exists.

Reply:
https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/config/index.ts